<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphEdit</class>
 <widget class="QWidget" name="GraphEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор графиков</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Тип графика</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rNone">
        <property name="text">
         <string>Нет</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rGraph">
        <property name="text">
         <string>График</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rHistro">
        <property name="text">
         <string>Гистрограмма</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layPrev">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Размеры</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Ширина:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinW">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2048</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Высота:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinH">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2048</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Шрифт:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinF">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblM">
          <property name="text">
           <string>Максимум:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinM">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>2048</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblImg">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
